config/detekt/detekt.yml

```diff
@@ -32,3 +32,6 @@ TwitterCompose:
   ComposableNaming:
     # TODO disabled for now pending https://github.com/twitter/compose-rules/issues/104
     active: false
+  CompositionLocalAllowlist:
+    active: true
+    allowedCompositionLocals: LocalTacoColorScheme
```

Comment:
Is there a better way to do this? We don't have to allow our existing composition locals here and I'm not clear why 😕.